config/locales/en.yml

```diff
@@ -2517,6 +2517,13 @@ en:
         edit:
           county: County
           title: Select the county where you lived on December 31, %{filing_year}
+      nj_household_rent_own:
+        edit:
+          helper_description: If you own a mobile home that is located in a mobile home park, you are considered a tenant.
```

Comment:
[dust] Why a helper description instead of in the tenant option like in the designs?

Reply:
Actually, in looking at the designs, seems like most of the content doesn't follow the Figma?
- title: You might be eligible for the New Jersey Property Tax credit or deduction!
- question label: In 2024, which were you?
- Option text is slightly different

Reply:
I followed the ticket acceptance criteria on this one. The design is still WIP, so there's a follow-up ticket coming soon to a GitHub project board near you